src/System Application/App/Agent/Setup/AgentList.Page.al

```diff
@@ -90,4 +90,14 @@ page 4316 "Agent List"
             }
         }
     }
+
+    trigger OnOpenPage()
```

Comment:
I think we also need to check if the capability is turned on. If there are agents but the capability is off, then we could raise a different notification.

Reply:
When they try to set up the agent, they'd get the dialog to enable the capability. I don't think we should change/move the user's flow away from that or add complexity.